README.md

```diff
@@ -33,8 +33,10 @@ We appreciate feedback and suggestions. If you would like certain features to be
    ```
 1. Start X-Plane and start a flight at an airport or your choice
 1. In the main menu, select **Plugins** -> **Air Traffic and Control** -> **Start World**
```

Comment:
Let's remove this line:
```
In the main menu, select **Plugins** -> **Air Traffic and Control** -> **Start World**
```
because it's outdated

Reply:
Yep. It's out of date.